test/e2e/suites/migrate-gitops/migrate_gitops_provv1_mgmtv3_test.go

```diff
@@ -23,12 +23,6 @@ import (
 	_ "embed"
 
 	. "github.com/onsi/ginkgo/v2"
-	"sigs.k8s.io/controller-runtime/pkg/envtest/komega"
-
-	"k8s.io/utils/ptr"
-
-	"github.com/rancher/turtles/test/e2e"
-	"github.com/rancher/turtles/test/e2e/specs"
```

Comment:
these should be needed and left as is

Reply:
Looks like my ide was messing with this for some reason. Thanks for spotting.